spec-dtslint/operators/elementAt-spec.ts

```diff
@@ -0,0 +1,22 @@
+import { of } from 'rxjs';
+import { elementAt } from 'rxjs/operators';
+
+it('should infer correctly', () => {
+  const o = of('foo').pipe(elementAt(47)); // $ExpectType Observable<string>
+});
+
+it('should support a default value', () => {
+  const o = of('foo').pipe(elementAt(47, 'bar')); // $ExpectType Observable<string>
+});
```

Comment:
I guess this could be like `startWith` and widen the type if a non-`T` default is specified. Do you want to create an issue/PR for that?